tests/similarity/test_pairs.py

```diff
@@ -96,6 +99,22 @@ def test_absolute_difference():
     assert_array_almost_equal(Xt, [[0, 0], [1, 1], [0, 0], [1, 1]])
 
 
+def test_CharacterEquality():
+    """Test for CharacterEquality."""
+    X = np.array(['q', 'q', 'q', '', '', 'q', 'a', '', '', 'q'])
```

Comment:
Hmmm, shouldnt you rather test with a 2d array? How can this work with 1d array? (Data is supposed to be "paired" data, with at least 2 features)